Product/SoapUI_Test/ValidationSuite/ConnectValidation-soapui-project.xml

```diff
@@ -4212,15 +4212,21 @@ context.withSql('DocRepositoryDB') { sql ->
 		}</script></con:config></con:testStep><con:setupScript>nhinc.FileUtils.backupConfiguration(context.findProperty('GatewayPropDir'), log);
 nhinc.FileUtils.setG1ConnectionInfo(context.expand(context.testCase.testSuite.project.resourceRoot), context.findProperty('GatewayPropDir'), log);</con:setupScript><con:tearDownScript>nhinc.FileUtils.restoreConfiguration(context.findProperty('GatewayPropDir'), log);</con:tearDownScript><con:properties><con:property><con:name>startDate</con:name><con:value>2018-06-20T13:19:14Z</con:value></con:property><con:property><con:name>endDate</con:name><con:value>2018-06-20T13:29:14Z</con:value></con:property><con:property><con:name>sigDate</con:name><con:value>06/20/2018 13:19:14</con:value></con:property><con:property><con:name>expireDate</con:name><con:value>2018-07-20T00:00:00Z</con:value></con:property></con:properties></con:testCase><con:testCase id="d48d01ba-daea-4bf7-a7e5-b17dfce72098" failOnError="true" failTestCaseOnErrors="true" keepSession="false" maxResults="0" name="Patient Location Query" searchProperties="true" disabled="true"><con:settings/><con:testStep type="groovy" name="clear-database on RequestPatientId" id="28b975e6-00b7-4a03-870b-b9241c7d4348"><con:settings/><con:config><script>log.info "info-log: delete-PatientCorrelation-RequestPatientID"
 context.withSql('PatientCorrelationDB') { sql -> 
-	def sqlDel  = 'delete from ' + context.findProperty('PatientCorrelationTable')+ ' where PatientAssigningAuthorityId="'+context.findProperty('LocalAA')+'" and PatientId="'+context.findProperty('PLQRequestPatientId')+'"'
+	def sqlDel  = 'delete from correlatedidentifiers where PatientAssigningAuthorityId="1.1" and PatientId="D123401"'
```

Comment:
Please change this to just clearing the correlations table. For this test, we want to start with a fresh table, not just clear a specific record.